repl/src/repl.rs

```diff
@@ -299,18 +300,24 @@ fn eval_line_(
             compiler = compiler.run_io(true);
             expr.run_expr(&mut compiler, vm, "line", line, None).boxed()
         }
-        Some(ReplLine::Let(let_binding)) => {
+        Some(ReplLine::Let(mut let_binding)) => {
```

Comment:
I like this new `ReplLine` enum, it will definitely make adding type expressions easier